examples/iowa.py

```diff
@@ -24,4 +24,7 @@
 }
 
 # Thumbnail Support
-THUMBNAIL = False
\ No newline at end of file
+THUMBNAIL = False
+
+# Google Analytics
+GA_ID = '103831764'
```

Comment:
calling {{ config.GA_ID }} on the whole id only returns UA. Any idea why this is happening?